docs/src/projects/configuration.md

```diff
@@ -105,6 +105,56 @@ response: [0x1, 0x23, 0x4]
 </details>
 <br>
 
+### Global configuration
+
+Global configuration file is usual [`snfoundry.toml`](https://foundry-rs.github.io/starknet-foundry/appendix/snfoundry-toml.html), 
+which is a common storage for configurations to apply to multiple projects across various directories.
+This file is stored in a predefined location and is managed differently compared to the local configuration file, which can be found in any parent directory.
+Profiles from global configurations are used to combine configuration with local profiles.
+
+#### Interaction between Local and Global profiles
+
+Interaction between global and local config is based on the overriding mechanism
+
+The local configuration will be used to override a global profile of the same name, or if no matching profile exists, it will override the global default profile. 
+If `--profile` is not explicitly specified, the default profiles from both the local and global configurations (if any is present) will be used.
+
+#### Global Configuration File Location
+The global configuration is stored in a specific location depending on the operating system:
+
+- macOS/Linux : The global configuration file is located at `$HOME/.config/starknet-foundry/snfoundry.toml`
+- Windows : The file can be found at `C:\Users\<user>\AppData\Roaming\starknet-foundry\snfoundry.toml`
+
+> 📝 **Note**
+> If missing, global configuration file will be created automatically on running any `sncast` command for the first time.
+
+### Config interaction example
+
+```
+root/
+├── .config/
+│   └── starknet-foundry/
+│       └── snfoundry.toml -> A
+└── /../../
+        └── projects/
+            ├── snfoundry.toml -> B
+            └── cairo-projects/
+                └── opus magnum/
+```
+
+**Glossary:**
+
+- **A:** Global configuration file containing the profiles `default` and `testnet`.
+- **B:** Local configuration file containing the profiles `default` and `mainnet`.
+
+In any directory in the file system, a user can run the `sncast` command using the `default` and `testnet` profiles, 
+because they are located in global config. 
+If no profiles are explicitly specified, the `default` profile from the global configuration file will be used.
+
+When running `sncast` from the `opus magnum` directory, there is a configuration file in the parent directory (B). 
+This setup allows for the use of the following profiles: `default`, `testnet`, and `mainnet`. If the `mainnet` profile is specified, 
+the configuration from the local file will be used to override the global `default` profile, as the `mainnet` profile does not exist in the global configuration.
```

Comment:
It's a nice example but maybe show a shortened contents of A and B instead?

Reply:
I've added links to proper sections, profile examples are defined above this section. I hope it will help avoid being redundant and also allow users to grab context. wdyt?